setup.cfg

```diff
@@ -31,6 +31,7 @@ install_requires =
     bids-validator
     num2words
     click
+    graphviz
```

Comment:
I'd rather not make this a requirement. We can add it as an extra.

Reply:
I put it under just test for now. Do you want me to make graphviz its own `extras_require`?

Reply:
Yes, let's just call it `plotting`, and also add `%(plotting)s` to `ci_tests`.